### ISSUE TITLE
gitignored .env files do not show up in info log

### ISSUE THREAD

**Issue by Roesh** [blitz-js/blitz#2333]
### What is the problem?

When running commands that display which .env files are loaded, we get the following logs:

info  - Loaded env from `<env file name>`
info  - Loaded env from `<env file name 2>`

By default, .gitignore ignores the following files:
```
.env.local
.env.*.local
.envrc
```

Running `blitz dev` under this rule I see
```
ready - started server on 0.0.0.0:3000, url: http://localhost:3000
info  - Loaded env from C:\Users\Roshan\Documents\DeleteMe\testapp\.env
info  - Using webpack 4. Reason: future.webpack5 option not enabled https://nextjs.org/docs/messages/webpack5
```

If I then comment out the gitignores and retry, I get 
```
ready - started server on 0.0.0.0:3000, url: http://localhost:3000
info  - Loaded env from C:\Users\Roshan\Documents\DeleteMe\testapp\.env.local
info  - Loaded env from C:\Users\Roshan\Documents\DeleteMe\testapp\.env      
info  - Using webpack 4. Reason: future.webpack5 option not enabled https://nextjs.org/docs/messages/webpack5
```

Note that in either case, if an environment variable is present in .env.local it is picked up, even though we don't get a logged message that .env.local is included, so in this scenario, it is a purely visual discrepancy.

---------------------------------------------------- 

Context: In my production server, I have a .env.production file, and it will show up in the logs when I run yarn build. However, I wanted to do the production build on my local environment since the prod server would hang up because the build step was so heave, so I copied that file to my dev environment, and added a gitignore line to ignore .env.production so that I don't accidentally check it in, and I found that it was never picked up. 

I am actually unsure if the values I add to it are being picked up, unlike the local case, but that will be a separate issue with separate testing

### Paste all your error logs here:

```
PASTE_HERE (leave the ``` marks)
```

### Paste all relevant code snippets here:

```
PASTE_HERE (leave the ``` marks)
```

### What are detailed steps to reproduce this?

1. install the latest blitz version
2. Create a new blitz app
3. run yarn dev, observe the output
4. comment out the .env lines in .gitignore
5. run yarn dev again

### Run `blitz -v` and paste the output here:

```
Windows 10 | win32-x64 | Node: v15.8.0

blitz: 0.34.3 (global)
blitz: 0.34.3 (local)

  Package manager: yarn
  System:
    OS: Windows 10 10.0.19042
    CPU: (16) x64 AMD Ryzen 7 2700 Eight-Core Processor
    Memory: 6.51 GB / 15.93 GB
  Binaries:
    Node: 15.8.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.10 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 7.6.0 - C:\Program Files\nodejs\npm.CMD
    Watchman: Not Found
  npmPackages:
    @prisma/client: ~2.20 => 2.20.1
    blitz: 0.34.3 => 0.34.3
    prisma: ~2.20 => 2.20.1
    react: 0.0.0-experimental-6a589ad71 => 0.0.0-experimental-6a589ad71
    react-dom: 0.0.0-experimental-6a589ad71 => 0.0.0-experimental-6a589ad71
    typescript: ~4.2 => 4.2.4
```

**Comment by Roesh** [comment]
This log comes from [nextjs](https://github.com/blitz-js/blitz/blob/830e92e956f703f9cbe7053bb669439f403b1b3b/nextjs/packages/next-env/index.ts#L46)

I'm going to see if gitignored files not showing up is intentional in the nextjs docs, and work backwards from there about why the discrepancy exists

**Comment by Roesh** [comment]
Hmmmm, so I was fiddling around with the latest blitz version, and I don't see the behavior there... 0.35.0-canary.6 (local) I think I'll wait till that version comes and close this issue

**Comment by flybayer** [comment]
So although next.js supports loading `.env.local` files, we manually load those files using dotenv before invoking Next. The logs come from next.js which, as you said, doesn't include the gitignored files.

So those files are loading, just not being logged. This will be resolved once the next.js fork migration is complete
